xbmc/guilib/guiinfo/VideoGUIInfo.cpp

```diff
@@ -221,23 +221,19 @@ bool CVideoGUIInfo::GetLabel(std::string& value, const CFileItem *item, int cont
         int iSeason = -1, iEpisode = -1;
```

Comment:
`iSeason` now is unused in this branch

```suggestion
        int iEpisode = -1;
```

Reply:
thx! updated.